importers/src/main/groovy/whelk/importer/DatasetImporter.groovy

```diff
@@ -0,0 +1,126 @@
+package whelk.importer
+
+import jline.internal.Log
+import org.codehaus.jackson.map.ObjectMapper
+import whelk.Document
+import whelk.IdGenerator
+import whelk.Whelk
+import whelk.exception.CancelUpdateException
+
+class DatasetImporter {
+    static final ObjectMapper mapper = new ObjectMapper()
+
+    static void importDataset(Whelk whelk, String filePath, String dataset, String marcCollection) {
+
+        if ( ! ["auth", "bib", "hold", "definitions"].any{ it.equals(marcCollection) } )
```

Comment:
Introducing datasets is another step towards possibly removing our LDDB collection notion (or at least reduce it to an internal optimization). I assume we don't want to do that in one step, but at this point we reasonably don't want to allow datasets to be anything other than in "definitions" either (which today basically means "from a dataset" (and is close to "any record not in auth, bib, or hold", albeit I've gathered the abstract Works fell in between for auth export reasons)).

So I suggest hardcoding the "definitions" collection for datasets (and note that in the issue about removing collections).